docs/internal-documentation/maintenance/ReleaseProcess.md

```diff
@@ -5,51 +5,87 @@ Release process
  * zonemaster-backend - [Changes](https://github.com/dotse/zonemaster-backend/blob/master/CHANGES)
  * zonemaster-gui - [Changes](https://github.com/dotse/zonemaster-gui/blob/master/Changes)
 
 ## 3. Update prerequisites
 
-Make sure the prerequisites section in [README.md](https://github.com/dotse/zonemaster/blob/master/README.md)
-is up to date with regard to [SupportCriteria](https://github.com/dotse/zonemaster/blob/master/docs/internal-documentation/maintenance/SupportCriteria.md).
+Make sure the [declaration of prerequisites] is up to date with regard to
+[SupportCriteria](SupportCriteria.md).
 
-## 4. Verify that MANIFEST is up to date
+## 4. Update [CI] configuration
 
-In order to have a complete installation from a package, the MANIFEST needs
-to be the complete set of files to be included.
+Make sure the Travis configuration for each repo is up to date with the supported Perl versions.
 
- * zonemaster-engine - (it will be created by the `make manifest` command below)
- * zonemaster-cli - [MANIFEST](https://github.com/dotse/zonemaster-cli/blob/master/MANIFEST)
- * zonemaster-backend - [MANIFEST](https://github.com/dotse/zonemaster-backend/blob/master/MANIFEST)
- * zonemaster-gui - [MANIFEST](https://github.com/dotse/zonemaster-gui/blob/master/MANIFEST)
+ * zonemaster-ldns - [.travis.yml](https://github.com/dotse/zonemaster-ldns/blob/master/.travis.yml)
+ * zonemaster-engine - [.travis.yml](https://github.com/dotse/zonemaster-engine/blob/master/.travis.yml)
+ * zonemaster-cli - [.travis.yml](https://github.com/dotse/zonemaster-cli/blob/master/.travis.yml)
+ * zonemaster-backend - [.travis.yml](https://github.com/dotse/zonemaster-backend/blob/master/.travis.yml)
+ * zonemaster-gui - [.travis.yml](https://github.com/dotse/zonemaster-gui/blob/master/.travis.yml)
 
-## 5. Verify that Makefile.PL has all the correct data
+## 5. Verify that MANIFEST is up to date
+
+Perform the following for each component except Zonemaster::Engine.
+
+> **Note:** For the Zonemaster::Engine component, the MANIFEST file is not
+> included in the repo, but created dynamically in step 7 below. This is subject
+> to change.
+
+Make sure your working directory is clean, or that all listed changes are
+covered by MANIFEST.SKIP:
+
+    git status --ignored
+
+> **Note:** To throw away any and all changes to tracked and untracked files you
+> can run `git clean -dfx ; git reset --hard`.
+
+Generate Makefile, META.yml and others.
+
+    mkdir -p inc/.author
+    perl Makefile.PL
+
+> **Note**: [`inc/.author` is an indicator used by Module::Install], which is used
+> in all components but Zonemaster::GUI.
+
+For all components except Zonemaster::Engine, make sure that all files are
+covered by MANIFEST and/or MANIFEST.SKIP:
+
+    make distcheck
+
+For Zonemaster::Engine, generate a new MANIFEST:
+
+    make manifest
+
+Make sure the MANIFEST file includes exactly the files that are supposed to be
+included in the dist tarball.
```

Comment:
Should we really require inspection every time?

"The MANIFEST file lists the files that will be included in the dist tarball. Review the MANIFEST file if in doubt or if there seems to be missing or extra files."

Reply:
Accepted.